source/ExampleTrait.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: roman
+ * Date: 31.10.16
+ * Time: 1:15.
+ */
+
+namespace Roman\Parser;
+
+trait ExampleTrait
```

Comment:
ExampleTrait is a bad name, it doesn't tell what it is doing. Rename it for example to FileUploadTrait or so